### PR TITLE
Emergency checklists

### PR DESCRIPTION
**Pull request by dg-505**
Added some emergency checklists:
- Engine failure during takeoff roll
- Engine failure after takeoff
- Engine failure during flight
- Emergency landing without engine power
- Ditching (emergency water landing)

If someone with more nasal knowledge than me could implement engine failures or engine fire when overspeed or overheating, I can write a checklist regarding engine fire. As soon as this is done, in my opinion the systems deserve a 5 Star rating.

### CODE REVIEW

**Comment by onox** [comment]
Cool to have these checklists! :smile: 

1. Can you update the marker position of "Mixture" in the "Engine failure during takeoff roll", "Engine failure after takeoff", and "Landing without engine power"?

2. Could you also fix the marker position and size (increase it a bit) of "Mixture" in "Starting Engine"? Try to make it a bit more consistent with the marker of the throttle.

3. Please take a look at the "Primer" text of "Engine during flight" checklist. The GUI doesn't look right. Perhaps you can make the text a bit shorter and move some of the text to the line below it. See the "Starting Engine" checklist.

4. Fix "Radio" in the "Ditching" checklist as well.

5. For each checklist title, can you make each first character of each upper case? "Engine Failure" instead of "Engine failure". Then it is more consistent with the titles of the other checklists.

**Comment by dg-505** [comment]
Roger, onox, thanks for the hints. I'm gonna do it tomorrow :)

**Comment by dg-505** [comment]
@onox I uploaded the modifications:

1. The marker position was correct: It was at the same place where the mixture lever is when pulled full out.
2. Increased the marker position a bit
3. I split the text up into two lines
4. Same as in 3.
5. Done

**Comment by onox** [comment]
IMO the marker should be more where the mixture slider connects with the panel. More like how the throttle is. I'll test the other points when I land after the USA tour.

**Comment by onox** [comment]
Could you also increase the marker of "Mixture" in "Engine Failure During Flight"? Other issues seems to have been resolved :+1:

**Comment by dg-505** [comment]
Both of your queries done :+1:

**Comment by onox** [comment]
Thanks! :smile:

**Comment by wkitty42** [comment]
how do we fix the menu overflow?
fgfs-screen-049

**Comment by dg-505** [comment]
Don't know. Maybe make the title shorter (any suggestions?) or make the menu wider (how can this be done?)

**Comment by onox** [comment]
To make the menu wider you would need to edit `gui/dialogs/checklist.xml` in fgdata, line 336, change `<pref-width>230</pref-width>` to `<pref-width>300</pref-width>`. Submit a patch to the mailing list and hope that they accept it.

**Comment by wkitty42** [comment]
how about this pull request #258 ? i abbreviated and shortened to fit... looks ok?
fgfs-screen-050

**Comment by gilbertohasnofb** [comment]
Looks a bit confusing to me. What about leaving "EMERGENCY" full but shortening "Engine" to "Eng" on all four instances? I am not a native speaker but EMGCY looks quite funny and obscure to me.

**Comment by dg-505** [comment]
What about **EMERG: ENG: Fail during Takeoff Roll**

**Comment by wkitty42** [comment]
i checked for abbreviations of emergency... thought this looked best of the one i saw at http://www.allacronyms.com/emergency/abbreviated but there are others, too... the idea was to get it to fit in the existing space alloted and the full word was just too much even with the other shortening... "EMERG" is another option instead of "EMGCY"... they're both 5 characters but proportional fonts stink for decent alignments like we're looking for...

**Comment by gilbertohasnofb** [comment]
@dg-505 Looking at the screenshot above, by removing the `:` and switching `Engine` to `Eng` should be enough to fit `EMERGENCY` by full. 

I very much prefer:
`EMERGENCY Eng Failure Takeoff Roll`
to
`EMGCY: Engine Failure Takeoff Roll`
or
`EMERG: Eng Fail during Takeoff Roll`

**Comment by wkitty42** [comment]
there's three engine failure scenarios... hummm... maybe this...

EMERG: Eng. Fail (Takeoff Roll)
EMERG: Eng. Fail (Takeoff)
EMERG: Eng. Fail (Flight)
EMERG: Ditching

**Comment by gilbertohasnofb** [comment]
@wkitty42 why abbreviating so much? As long as the phrase fits inside the box then it should be fine, right?

**Comment by wkitty42** [comment]
right but it wasn't fitting in the box and i thought to make it ""like a bikini - short and to the point"" without being too wordy... let me look at it again and do another pic...

**Comment by gilbertohasnofb** [comment]
Okay, thanks for having a look at this! 

> like a bikini - short and to the point

:laughing:

**Comment by wkitty42** [comment]
how's this? it is close but it fits ok...
fgfs-screen-051

**Comment by gilbertohasnofb** [comment]
Looks much better to me. If you think it's too close you may consider removing the parenthesis around `(Takeoff Roll)`, or removing the `:` after `EMERGENCY`, but to be honest it looks pretty fine to me now, so it's up to you :smile:

**Comment by dg-505** [comment]
For the third maybe: EMERGENCY: Eng. Fail (Landing)

**Comment by wkitty42** [comment]
anyone else have any comments? if this is good, i just need to figure out how to send it up to my fork to replace the other one i already sent up and made the pull request with...

**Comment by wkitty42** [comment]
@dg-505 do you mean the 4th one? remove "w/o"? sure... i was kinda thinking about that if that's what you mean...

**Comment by dg-505** [comment]
Yeah, I meant exactly this

**Comment by gilbertohasnofb** [comment]
I agree with @dg-505, just Landing is better than Landing w/o (if the engine fails during landing, it's quite obvious it will be without engine :wink: ).

As for git, probably you should cancel your PR and make a new one.

**Comment by wkitty42** [comment]
ok... i'll remove "w/o", cancel the PR and send a new one...

**Comment by gilbertohasnofb** [comment]
Thanks a lot, really appreciated.

**Comment by wkitty42** [comment]
#260 is ready to go... no problem in the pedantism... i do that at times, too :wink: 
it is kinda my first contribution to the project instead of just reports, suggestions and testing... i'm glad you and @dg-505 were open to the suggestions and changes :smile_cat:

**Comment by dg-505** [comment]
Thats of course! Every single contribution that makes this project better is very welcome.

**Comment by wkitty42** [comment]
:+1:  thanks guys... it is a good feeling :angel:
